llama_index/vector_stores/pinecone.py

```diff
@@ -159,6 +160,26 @@ def _to_pinecone_filter(standard_filters: MetadataFilters) -> dict:
 )
 
 
+def apply_regex(input_string: str) -> str:
```

Comment:
@zackproser @DosticJelena, fyi had to add this bc otherwise the `__version__` (latest) would be "3.0.0.dev6", which wouldn't compare correctly with "3.0.0"